src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.CinemaHall;
+
+public interface CinemaHallDao {
+    CinemaHall add(CinemaHall cinemaHall);
+
+    Optional<CinemaHall> get(Long id);
```

Comment:
```suggestion
    CinemaHall get(Long id);
```
create CinemaHallDaoImpl and CinemaHallServiceImpl. The service layer should have the following methods:
  
```
  CinemaHall add(CinemaHall cinemaHall);

    CinemaHall get(Long id);

    List<CinemaHall> getAll();
```

Reply:
agree